go/test/endtoend/cluster/vttablet_process.go

```diff
@@ -109,52 +124,109 @@ func (vttablet *VttabletProcess) Setup() (err error) {
 		vttablet.exit <- vttablet.proc.Wait()
 	}()
 
-	timeout := time.Now().Add(60 * time.Second)
-	for time.Now().Before(timeout) {
-		if vttablet.WaitForStatus("NOT_SERVING") {
-			return nil
-		}
-		select {
-		case err := <-vttablet.exit:
-			return fmt.Errorf("process '%s' exited prematurely (err: %s)", vttablet.Name, err)
-		default:
-			time.Sleep(300 * time.Millisecond)
-		}
+	err = vttablet.WaitForTabletType(vttablet.ServingStatus)
+	if err != nil {
+		return fmt.Errorf("process '%s' timed out after 60s (err: %s)", vttablet.Name, <-vttablet.exit)
+	}
+	return nil
+}
+
+// GetTabletStatus function checks if vttablet process is up and running
+func (vttablet *VttabletProcess) GetTabletStatus() string {
+	resultMap := vttablet.GetVars()
+	if resultMap != nil {
+		return reflect.ValueOf(resultMap["TabletStateName"]).String()
+	} else {
+		return ""
 	}
+}
 
-	return fmt.Errorf("process '%s' timed out after 60s (err: %s)", vttablet.Name, <-vttablet.exit)
+// GetStatus function checks if vttablet process is up and running
+func (vttablet *VttabletProcess) GetStatus() string {
```

Comment:
What is the difference between this and `GetTabletStatus`? Why do we need both?

Reply:
`GetTabletStatus` gets the tablet status, but `GetStatus` hits `/debug/status` gets html page, where we have some assertion regarding the binlog player.

Reply:
Can you change the comment describing these two functions?